public/javascripts/audit_log.js

```diff
@@ -0,0 +1,18 @@
+function setup_audit_table()
+{
```

Comment:
as Ancor protested so loudly last time he actually reviewed - we're moving towards proper javascript names. Which are camel case